### PR TITLE
remove grey button from approver

### PR DESCRIPTION
**Pull request by mugnivenko**

### CODE REVIEW

File: pages/care-package/home-care/approve-brokered/[id].js [hunk 1]
```diff
@@ -66,11 +68,33 @@ const HomeCareApproveBrokered = () => {
...
+
+  const handleRequestMoreInformation = () => {
+    const { validFields, hasErrors } = fieldValidator([{
+      name: 'requestInformationText', value: requestInformationText, rules: ['empty'],
+    }]);
+
+    setErrorFields(validFields);
+
+    if(hasErrors) return;
+    console.log('request more information info');
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-brokered/[id].js:95]: remove console log

File: pages/care-package/home-care/approve-package/[id].js [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
-import { getServiceTypeCareTimes } from '../../../../service/homeCareServiceHelper';
-import useHomeCareApi from '../../../../api/SWR/useHomeCareApi'
-import { Button } from '../../../../components/Button'
-import ClientSummaryItem from '../../../../components/CarePackages/ClientSummaryItem'
-import TitleHeader from '../../../../components/TitleHeader'
-import DaySummary from '../../../../components/HomeCare/DaySummary'
-import { testDataDaySummaries } from '../../../../testData/testDateHomeCare'
+import { getErrorResponse, getUserSession } from '../../../../service/helpers';
+import useHomeCareApi from '../../../../api/SWR/useHomeCareApi';
+import { Button } from '../../../../components/Button';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:10]: change import to 'components/Button';

File: pages/care-package/home-care/approve-package/[id].js [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
+import ClientSummaryItem from '../../../../components/CarePackages/ClientSummaryItem';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:11]: change import to 'components/..........'

File: pages/care-package/home-care/approve-package/[id].js [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
+import TitleHeader from '../../../../components/TitleHeader';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:12]: change import to 'components/..........'

File: pages/care-package/home-care/approve-package/[id].js [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
+import DaySummary from '../../../../components/HomeCare/DaySummary';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:13]: change import to 'components/..........'

File: pages/care-package/home-care/approve-package/[id].js [hunk 6]
```diff
@@ -1,19 +1,23 @@
...
+import { testDataDaySummaries } from '../../../../testData/testDateHomeCare';
+import RequestMoreInformation from '../../../../components/Approver/RequestMoreInformation';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:15]: change import to 'components/..........'

File: pages/care-package/home-care/approve-package/[id].js [hunk 7]
```diff
@@ -1,19 +1,23 @@
...
+import fieldValidator from '../../../../service/inputValidator';
+import { nursingCareRequestClarification } from '../../../../api/CarePackages/NursingCareApi';
+import { APPROVER_HUB_ROUTE } from '../../../../routes/RouteConstants';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:18]: change import to 'routes/..........'

File: pages/care-package/home-care/approve-package/[id].js [hunk 8]
```diff
@@ -1,19 +1,23 @@
...
+import { getErrorResponse, getUserSession } from '../../../../service/helpers';
+import useHomeCareApi from '../../../../api/SWR/useHomeCareApi';
+import { Button } from '../../../../components/Button';
+import ClientSummaryItem from '../../../../components/CarePackages/ClientSummaryItem';
+import TitleHeader from '../../../../components/TitleHeader';
+import DaySummary from '../../../../components/HomeCare/DaySummary';
+import { testDataDaySummaries } from '../../../../testData/testDateHomeCare';
+import RequestMoreInformation from '../../../../components/Approver/RequestMoreInformation';
+import fieldValidator from '../../../../service/inputValidator';
+import { nursingCareRequestClarification } from '../../../../api/CarePackages/NursingCareApi';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:17]: change import to 'api/..........'

File: pages/care-package/home-care/approve-package/[id].js [hunk 9]
```diff
@@ -1,19 +1,23 @@
...
-import { PERSONAL_CARE_MODE } from '../../../../service/homeCarePickerHelper';
-import { getServiceTypeCareTimes } from '../../../../service/homeCareServiceHelper';
-import useHomeCareApi from '../../../../api/SWR/useHomeCareApi'
-import { Button } from '../../../../components/Button'
-import ClientSummaryItem from '../../../../components/CarePackages/ClientSummaryItem'
-import TitleHeader from '../../../../components/TitleHeader'
-import DaySummary from '../../../../components/HomeCare/DaySummary'
-import { testDataDaySummaries } from '../../../../testData/testDateHomeCare'
+import { getErrorResponse, getUserSession } from '../../../../service/helpers';
+import useHomeCareApi from '../../../../api/SWR/useHomeCareApi';
```
Comment (akeamwoodsND) [inline pages/care-package/home-care/approve-package/[id].js:9]: change import to 'api/..........'

File: components/packages/day-care/index.js [hunk 10]
```diff
@@ -13,6 +13,11 @@ import DayCareSummary from '../../DayCare/DayCareSummary';
 import ProposedPackagesTab from '../ProposedPackagesTabs';
 import { formatCareDatePeriod } from '../../../service/helpers'
 import ClientSummaryItem from '../../CarePackages/ClientSummaryItem'
+import PopupAddSupplier from '../../PopupAddSupplier'
+import AutocompleteSelect from '../../AutocompleteSelect'
+import useSuppliersApi from '../../../api/SWR/useSuppliersApi'
```
Comment (akeamwoodsND) [inline components/packages/day-care/index.js:18]: 'api/swr........'

File: components/packages/day-care/index.js [hunk 11]
```diff
@@ -13,6 +13,11 @@ import DayCareSummary from '../../DayCare/DayCareSummary';
...
+import useDayCareApi from '../../../api/SWR/useDayCareApi'
```
Comment (akeamwoodsND) [inline components/packages/day-care/index.js:19]: same as above

File: components/packages/day-care/index.js [hunk 12]
```diff
@@ -13,6 +13,11 @@ import DayCareSummary from '../../DayCare/DayCareSummary';
...
+import { mapCareStageOptions } from '../../../api/Mappers/CarePackageMapper'
```
Comment (akeamwoodsND) [inline components/packages/day-care/index.js:20]: same as above

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Screenshot 2021-09-02 at 09 54 28

We have these aliases, so any import from components, constants, routes or api that looks like '.../.../.../components' can be shortened to 'components/...' please check this.

**Review by dunnudge — APPROVED** [review]

**Pull request merged**